### ISSUE TITLE
Set up your domain (host-specific instructions) ... contains no instructions

### ISSUE THREAD

**Issue by SetTrend** [MicrosoftDocs/microsoft-365-docs#4748]
The mentioned page is supposed to provide host-specific instructions ‒ but it's quite empty.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 34ecaa80-9bd4-0f65-ea64-3bcf9d4d0acb
* Version Independent ID: 6575fb2b-3e43-8095-c541-5860d2513f02
* Content: [Set up your domain (host-specific instructions) - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/get-help-with-domains/set-up-your-domain-host-specific-instructions?view=o365-worldwide)
* Content Source: [microsoft-365/admin/get-help-with-domains/set-up-your-domain-host-specific-instructions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/get-help-with-domains/set-up-your-domain-host-specific-instructions.md)
* Service: **o365-administration**
* GitHub Login: @pebaum
* Microsoft Alias: **pebaum**

**Comment by msbemba** [comment]
@SetTrend Thank you for your feedback. I'm working with the author to get the data updated.

Thanks
Sri

**Comment by cmcatee-MSFT** [comment]
@SetTrend We no longer maintain a list of steps for other DNS providers, so the links were removed. I've removed "(host-specific instructions)" from the title, and the article will be retired shortly. For instructions, see [Change nameservers to set up Microsoft 365 with any domain registrar](https://docs.microsoft.com/en-us/microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar?view=o365-worldwide).
